index.html

```diff
@@ -1,51 +1,37 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Deanna Nguyen | Product Designer</title>
-    <link rel="shortcut icon" type="image/x-icon" href="images/favicon.png"/>
-
-    <!-- BOOTSTRAP -->
-    <link href="css/bootstrap.min.css" rel="stylesheet">
-    <link rel="stylesheet" href="font-awesome-4.4.0/css/font-awesome.min.css">
-    <link rel="stylesheet" type="text/css" href="css/styles.css">
-    <link rel="stylesheet" type="text/css" href="css/desktop.css">
-    <link rel="stylesheet" type="text/css" href="css/laptops.css">
-    <link rel="stylesheet" type="text/css" href="css/landscapephones.css">
-    <link rel="stylesheet" type="text/css" href="css/portraitphones.css">
-  </head>
-
-  <body>
-  	<nav class="navbar sticky-top navbar-expand-sm">
-		<div class="container">
-			<!-- BRAND -->
-			<a class="navbar-brand" href="http://deannanguyen.com">DEANNA</a>
-
-  			<!-- LINKS -->
-	  		<div class="collapse navbar-collapse justify-content-end">
-		  		<ul class="navbar-nav">
-		  			<li class="nav-item">
-		  				<a class="nav-link" href="#">LINK ONE</a>
-		  			</li>
-		  		</ul>
-
-		  		<ul class="navbar-nav">
-		  			<li class="nav-item">
-		  				<a class="nav-link" href="#">LINK TWO</a>
-		  			</li>
-		  		</ul>
-
-		  		<ul class="navbar-nav">
-		  			<li class="nav-item">
-		  				<a class="nav-link" href="#">LINK THREE</a>
-		  			</li>
-		  		</ul>
-		  	</div>
-		</div>
-  	</nav>
-  </body>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Deanna Nguyen | Product Designer</title>
+        <link rel="shortcut icon" type="image/x-icon" href="images/favicon.png"/>
 
+        <!-- BOOTSTRAP -->
+        <link href="css/bootstrap.min.css" rel="stylesheet">
+        <link rel="stylesheet" type="text/css" href="css/styles.css">
+    </head>
 
+    <body>
+        <div id="app">
+            <nav class="navbar navbar-dark bg-dark sticky-top navbar-expand-sm">
+                <div class="container">
+                    <!-- BRAND -->
+                    <a class="navbar-brand" href="#">DEANNA</a>
```

Comment:
Adding button on line 21 is what you were missing. That is the major reason why the hamburger menu wasn't showing. However, even with this addition, the hamburger menu wouldn't appear. Adding the 'theme' classes of 'navbar-dark bg-dark' on line 17 ultimately resolved the issue.